modules/machine-health-checks-about.adoc

```diff
@@ -6,15 +6,32 @@
 [id="machine-health-checks-about_{context}"]
 = About machine health checks
 
-Machine health checks automatically repair unhealthy machines in a particular machine pool.
+You can define conditions under which machines in a Cluster are considered unhealthy by using a `MachineHealthCheck`.
+Machines matching the conditions are automatically remediated.
 
-To monitor machine health, you create a resource to define the configuration for a controller. You set a condition to check for, such as staying in the `NotReady` status for 15 minutes or displaying a permanent condition in the node-problem-detector, and a label for the set of machines to monitor.
+When defining a machine health check, you can specify conditions to check for and set a timeout for each of the conditions.
+A machine will be remediated if any of the conditions are met for the duration of the timeout.
+The action of remediating a machine will trigger a new machine to be created, to replace the failed one.
+
+For example, you may set a condition such as staying in the `NotReady` state for 15 minutes or displaying a permanent condition in the node-problem-detector.
 
 [NOTE]
 ====
-You cannot apply a machine health check to a machine with the master role.
+Please note that `MachineHealthChecks` currently *only* support machines that are owned by a `MachineSet`.
+Please review the <<machine-health-checks-limitations_{context}>> below for full details of machine health check limitations.
 ====
 
-The controller that observes a `MachineHealthCheck` resource checks for the status that you defined. If a machine fails the health check, it is automatically deleted and a new one is created to take its place. When a machine is deleted, you see a `machine deleted` event. To limit disruptive impact of the machine deletion, the controller drains and deletes only one node at a time. If there are more unhealthy machines than the `maxUnhealthy` threshold allows for in the targeted pool of machines, remediation stops so that manual intervention can take place.
+When a machine is deleted, you will see a `machine deleted` event.
+Remediation stops if there are more unhealthy machines in the pool than the `maxUnhealthy` threshold allows for.
+Manual intervention is required when remediation is halted.
+
+[id="machine-health-checks-limitations_{context}"]
+== Limitations when deploying MachineHealthChecks
+
+There are limitations to consider before deploying a MachineHealthCheck:
 
-To stop the check, you remove the resource.
+* Only machines owned by a MachineSet are remediated by a MachineHealthCheck.
+* Control plane machines are not currently supported and are not remediated if they are unhealthy.
+* If the node for a machine is removed from the cluster, a MachineHealthCheck considers the machine to be unhealthy and remediates it immediately.
+* If the machine's corresponding node does not join the cluster after the `nodeStartupTimeout`, the machine is remediated.
```

Comment:
Please change this to "If the corresponding node for a machine...", to avoid using a possessive.